Src/Generic/Viewables/Button.swift

```diff
@@ -15,13 +15,16 @@ import UIKit
 enum ButtonType {
     case standard(backgroundColor: HedvigColor, textColor: HedvigColor)
     case pillTransparent(backgroundColor: HedvigColor, textColor: HedvigColor)
+    case iconTransparent(textColor: HedvigColor, icon: ImageAsset?)
 
     func backgroundOpacity() -> CGFloat {
         switch self {
         case .standard:
             return 1
         case .pillTransparent:
             return 0.6
+        default:
```

Comment:
Avoid using default here, use the `.iconTransparent` case instead, also applies to the other ones.